lms/djangoapps/courseware/config.py

```diff
@@ -0,0 +1,19 @@
+"""
+Waffle flags and switches
+"""
+
+from edx_toggles.toggles import WaffleSwitch
+
+# Namespace
+WAFFLE_NAMESPACE = 'courseware'
+
+# .. toggle_name: courseware.enable_new_financial_assistance_flow
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: enables new financial assistance flow, when active.
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2022-03-25
+# .. toggle_tickets: https://openedx.atlassian.net/browse/PROD-2588
+ENABLE_NEW_FINANCIAL_ASSISTANCE_FLOW = WaffleSwitch(
+    f"{WAFFLE_NAMESPACE}.enable_new_financial_assistance_flow", __name__
+)
```

Comment:
We are not using WaffleSwitch. We will use Waffle Course override to allow selective courses activation of the new flow.